episodes/2-understanding-v6.md

```diff
@@ -0,0 +1,147 @@
+:::
+
+# Unique selling points of vantage6
+
+vantage6 is a platform to perform  privacy preserving federated learning. Other platforms for federated learning are current available, but vantage6 provides some unique features:
+-   Open source.
+-   Container orchestration for privacy enhancing techniques.
+-   Easily extensible to different types of data sources.
+-   Algorithms can be developed in any language.
+-   Other applications can connect to vantage6 using the API.
+
+
+# The vantage6 infrastructure
+
+In vantage6, a **client** can pose a question to the central **server**. Each organization with sensitive data contributes one **node** to the network. The nodes collects the research question from the server and fetches the **algorithm to answer** it. When the algorithm completes, the node sends the aggregated results back to the server.
+
+The roles of these vantage6 components are as follows:
+
+-   A (central) **server** coordinates communication with clients and nodes. The server tracks the status of the computation requests and handles administrative functions such as authentication and authorization.
+
+-   **Node(s)** have access to data and execute algorithms
+
+-   **Clients** (i.e. users or applications) request computations from the nodes via the client
+
+-   **Algorithms** are scripts that are run on the sensitive data. Each algorithm is packaged in a Docker image; the node pulls the image from a Docker registry and runs it on the local data. Note that the node owner can control which algorithms are allowed to run on their data.
+
+![v6 basic schema.](fig/v6_basic_schema.svg)
+
+On a technical level, vantage6 may be seen as a (Docker) container orchestration tool for privacy preserving analyses. It deploys a network of containerized applications that together ensure insights can be exchanged without sharing record-level data.
+
+## The vantage 6 infrastructure components
+
+As we saw before, the vantage6 network consists of a central server, a number of nodes and a client. This section explains in some more detail what these network actors are responsible for, and which subcomponents they contain.
+
+### Server
+
+The vantage6 server is the central point of contact for all communication in vantage6. However, it also relies on other infrastructure components to function properly. It consists of:
+
+-   **Vantage6 server**: Contains the users, organizations, collaborations, tasks and their results. It handles authentication and authorization to the system and is the central point of contact for clients and nodes.
+
+-   **Docker registry**: Contains algorithms stored in images which can be used by clients to request a computation. The node will retrieve the algorithm from this registry and execute it.
+
+-   **Algorithm store**: Is intended to be used as a repository for trusted algorithms within a certain project. Algorithm stores can be coupled to specific collaborations or to all collaborations on a given server.
+

```

Comment:
Not sure if we want to mention these. I think it is good to mention them briefly, even though the users do not really interact with them. 
```suggestion
- **EduVPN instance**
- **RabbitMQ**
```